os/arch/arm/src/armv8-m/up_svcall.c

```diff
@@ -531,6 +549,41 @@ int up_svcall(int irq, FAR void *context, FAR void *arg)
 	break;
 #endif
 
+#ifdef CONFIG_ARMV8M_TRUSTZONE
+	case SYS_init_securecontext: {
+		/* De-prioritize the non-secure exceptions so that the
+		 * non-secure pendSV runs at the lowest priority
+		 */
+		SecureInit_DePrioritizeNSExceptions();
```

Comment:
Could you let me know the location of prototypes and definitions for Securexxx functions?

Reply:
the definitions are in Secure world and are exposed as part of cmse_implib
please check secure component of RTK SDK OS

Reply:
Even though they are in library, we should include the header for prototype. I think it gives warnings.